corehq/apps/accounting/migrations/0064_add_form_link_workflow_priv.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.16 on 2022-12-20 06:20
+
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.privileges import FORM_LINK_WORKFLOW
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _grandfather_form_link_workflow_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        FORM_LINK_WORKFLOW,
+        skip_edition='Paused,Community,Standard',
+        noinput=True,
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0063_replace_linked_projects_ff_with_erm'),
+    ]
+
+    operations = [
+        migrations.RunPython(_grandfather_form_link_workflow_privs),
```

Comment:
If I'm not mistaken, this migration is not revertable if a "down" method is not specified. That is if we want it to be revertable

Reply:
We don't usually add a down method for these sort of migrations,